### ISSUE TITLE
Denoising with manual classification is not transparent enough

### ISSUE THREAD

**Issue by eurunuela** [ME-ICA/tedana#848]
### Summary

We should make [denoising with the manual classification](https://tedana.readthedocs.io/en/stable/usage.html#Arguments%20for%20Rerunning%20the%20Workflow) more transparent so that users know exactly what tedana is doing. Currently, tedana does not update the component figures (it should probably output a new folder with new images), nor does it say that the manual classification is being used to denoise the data. This can be confusing for users, so we should probably be more explicit on the prints/logs.

### Additional Detail

From a couple of emails with a tedana user:

> the png's in the 'figures' folder created by tedana appear to be unchanged (e.g., a component that was accepted via RICA is still shown as rejected).
The 'tedana.tsv' does state 'Using supplied mixing matrix from ICA', but there is no mention that a provided manual classification is used as well.
However, quickly eyeballing some voxel timecourses of the resulting 'desc-optcomDenoised_bold.nii' and comparing it to the pre-RICA version of the same file does seem to suggest some changes did occur (but it's kind of difficult to gauge).
It would be great to have this step work somewhat more transparent so users can be confident the updated denoising was actually implemented.

### Next Steps

- Make prints/logs more informative.
- Generate new folder of component figures?

**Comment by tsalo** [comment]
I agree that we could always improve our documentation, although I'm not sure about the specific issues this person reported. Is it possible they were using an older version of tedana?

>The 'tedana.tsv' does state 'Using supplied mixing matrix from ICA', but there is no mention that a provided manual classification is used as well.

We actually do log that manual classification is being performed in both the logger (which feeds to the tedana.tsv file) and the boilerplate report.

https://github.com/ME-ICA/tedana/blob/eaa0920f9f45f4eea0e2ab5f9e482e4d7ca3f324/tedana/selection/tedica.py#L39-L44

>the png's in the 'figures' folder created by tedana appear to be unchanged (e.g., a component that was accepted via RICA is still shown as rejected).

I took a look at the artifacts of the five-echo integration test in #849, where the workflow is run once regularly, and then run a second time (with a different output directory), with manacc specified according to the accepted components from the first run. While we can't check that manacc changed anything since the accepted components are the same across those two runs, all ignored components in the regular run are automatically classified as rejected when manacc is provided, so I was able to see that ignored components in one folder were labeled and colored appropriately as rejected components in the other.

## From the regular decision tree (component is ignored)

![image](https://user-images.githubusercontent.com/8228902/155231401-981ce622-0988-4363-a294-0ce3916d1966.png)


## From the manual classification (component is rejected)

![image](https://user-images.githubusercontent.com/8228902/155231431-8201c5a0-d81a-4a34-9706-1384bbab3d5e.png)

**Comment by eurunuela** [comment]
We might want to add some checks. It's the easiest way to rerun tedana after Rica.

**Comment by jreithler** [comment]
Thanks both for looking into this!

The test referred to above was done using tedana v0.0.11, but indeed without --manacc, as I figured the RICA *.tsv already included the updated component classifications. The resulting tedana *.tsv then does state that the supplied mixing matrix is used, but as far as I could see does not report that the manually classified components are taken into account. Similarly, the figures did not seem to get updated.

Maybe it would be helpful to e.g. create a subfolder in which only the updated component *.png's are saved when re-running the denoising based on manual classifications? This would allow to easily verify the desired changes were implemented without storing unnecessary duplicate figures for any unchanged components.
